packages/blade/src/components/Carousel/Indicators/Indicators.tsx

```diff
@@ -13,14 +13,20 @@ const Indicators = (props: IndicatorsProps): React.ReactElement => {
       {...makeAccessible({ role: 'tablist', label: 'Slides' })}
     >
       {new Array(props.totalItems).fill(0).map((_, idx) => {
+        const isResponsive = visibleItems === 'autofit';
+        let _visibleItems = visibleItems as 1 | 2 | 3;
```

Comment:
What is this anti-anurag anurag behaviour 😆?

Reply:
@anuraghazra be like

![image](https://github.com/razorpay/blade/assets/2913308/a9280b36-6c78-4b88-a7a3-52d52160bb4d)